modules/openapi-generator/src/test/resources/3_0/kotlin/issue7325-use-delegate-reactive-tags-kotlin.yaml

```diff
@@ -0,0 +1,35 @@
+openapi: "3.0.1"
+info:
+  title: test
+  version: "1.0"
+
+paths:
+
+  /api/v1/test/{id}:
+    get:
+      tags: [Test v1]
+      operationId: test1
+      parameters:
+        - name: id
+          in: query
```

Comment:
@de1mos242 thanks for the PR. Should it be a path parameter instead?

Reply:
I also tested this spec with the latest master and the output compiles without issues using `gradle build`. Is that supposed to fail to compile?

Reply:
@wing328 thanks for response. Yes, it should be a path, my mistake. 
About working master - I've just run on the latest master my test case and it generates this code:
`@Validated
@RequestMapping("\${api.base-path:}")
interface TestV1Api {

    fun getDelegate(): TestV1ApiDelegate = object: TestV1ApiDelegate {}


    @GetMapping(
            value = ["/api/v1/test/{id}"]
    )
    suspend fun test1( @PathVariable("id") id: kotlin.String
): ResponseEntity<Unit> {
        return getDelegate().test1(id, exchange);
    }
}`

There is an `exchange` variable used in return statement that hasn't been declared. I don't know how it could be compiled withtout errors

Reply:
In the latest master, can you try the following command?
```
java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g kotlin-spring -i https://raw.githubusercontent.com/OpenAPITools/openapi-generator/902a3c9349fcd76671d964f55fa4b259e595abab/modules/openapi-generator/src/test/resources/3_0/kotlin/issue7325-use-delegate-reactive-tags-kotlin.yaml -o /tmp/kotlin/ --skip-validate-spec
```

The output builds fine for me.
```
------------------------------------------------------------
Gradle 5.2.1
------------------------------------------------------------

Build time:   2019-02-08 19:00:10 UTC
Revision:     f02764e074c32ee8851a4e1877dd1fea8ffb7183

Kotlin DSL:   1.1.3
Kotlin:       1.3.20
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          1.8.0_282 (AdoptOpenJDK 25.282-b08)
OS:           Mac OS X 10.16 x86_64
```

Reply:
Looks like that you try to generate api with default settings and yes, it works fine. But I need to generate api with reactive stack using delegatePattern and tags.
Try to run 
`java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g kotlin-spring -i https://raw.githubusercontent.com/OpenAPITools/openapi-generator/902a3c9349fcd76671d964f55fa4b259e595abab/modules/openapi-generator/src/test/resources/3_0/kotlin/issue7325-use-delegate-reactive-tags-kotlin.yaml -o /tmp/kotlin/ --skip-validate-spec -p delegatePattern=true,useTags=true,reactive=true`

And open file `/tmp/kotlin/src/main/kotlin/org/openapitools/api/TestV1Api.kt`

Reply:
Thanks for the additional info. I'm able to repeat the issue and confirm your PR address fixes the issue.